.env.rb

```diff
@@ -0,0 +1 @@
+ENV={account_sid:"AC3b656407e6b3c247e8841404773378f5", auth_token:"bb97b237956971bdb3deb13c91763076", phone_number:"447908605707"}
```

Comment:
Metrics/LineLength: Line is too long. [130/80]<br>Style/FileName: Use snake_case for source file names.